test/conftest.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+import pytest
+
+from ThreeDiToolbox.datasource.netcdf_groundwater import NetcdfGroundwaterDataSource
+from ThreeDiToolbox.utils.patched_threedigrid import (
+    GridH5ResultAdmin,
+    GridH5AggregateResultAdmin,
+)
+
+
+cur_dir = os.path.dirname(__file__)
+data_dir = os.path.join(cur_dir, "data")
```

Comment:
Ik zat er over te denken of ik dit (als zijnde constantes) nou met hoofdletters zou willen of niet.
Ik kan me de keuze voor kleine letters wel voorstellen. Het is meer initialisatie dan dat het nou een echte constante is.

Was de keuze bewust?

Reply:
Nee, geen bewuste keuze.